script.js

```diff
@@ -0,0 +1,27 @@
+// one-var
+
+var a = 10;
+
+const value = 10;
+value++;
+
+//comment whitout space
+
+const func = function(){
+
+  let ad = 10
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['ad' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/WebHeroSchool/js_test/pullRequest?prid=5502397)